### PR TITLE
[Improvement] Fix the flaky test

### PR DESCRIPTION
**Pull request by jerqi**
### What changes were proposed in this pull request?
Fix the flaky test.

### Why are the changes needed?
Flaky test will cause ga fail sometimes.

### Does this PR introduce _any_ user-facing change?
No

### How was this patch tested?
No, test only.

### CODE REVIEW

File: integration-test/common/src/test/java/com/tencent/rss/test/HealthCheckCoordinatorGrpcTest.java [hunk 1]
```diff
@@ -129,9 +132,16 @@ public void healthCheckTest() throws Exception {
     assertEquals(ResponseStatusCode.INTERNAL_ERROR, response.getStatusCode());
 
     tempDataFile.delete();
-    Uninterruptibles.sleepUninterruptibly(3, TimeUnit.SECONDS);
-    nodes  = coordinators.get(0).getClusterManager()
-        .getServerList(Sets.newHashSet(Constants.SHUFFLE_SERVER_VERSION));
+    int i = 0;
+    do {
```
Comment (colinmjj) [inline integration-test/common/src/test/java/com/tencent/rss/test/HealthCheckCoordinatorGrpcTest.java:136]: can the retry strategy be replaced with CoordinatorTestUtils.waitForRegister?

Comment (jerqi) [reply integration-test/common/src/test/java/com/tencent/rss/test/HealthCheckCoordinatorGrpcTest.java:136]: We fetch the ServerNode objects, not only the number of the servers. So it cannot be replaced.

**Comment by colinmjj** [comment]
LGTM

**Comment by jerqi** [comment]
@colinmjj Do we merge it?

**Review by colinmjj — APPROVED** [review]

**Pull request merged**